art-gallery-medium-article-website/config-overrides.js

```diff
@@ -13,7 +13,7 @@ module.exports = function override(config) {
   config.resolve.fallback = fallback;
   config.plugins = (config.plugins || []).concat([
     new webpack.ProvidePlugin({
-      process: "process/browser",
+      process: "process/browser.js",
```

Comment:
@micwallace is this intentional to add .js to this? I think this is a typo.

Reply:
Yeah it's fixed the issue with Magic and probably a few other es modules that use .mjs extension. Similar issue here: 
https://github.com/graphql/graphql-js/issues/2721